src/constants.js

```diff
@@ -1,5 +1,12 @@
-import { modes as originalModes } from '@mapbox/mapbox-gl-draw/src/constants';
-export { cursors, geojsonTypes, events, meta, activeStates } from '@mapbox/mapbox-gl-draw/src/constants';
+import MapboxDraw from '@mapbox/mapbox-gl-draw';
+const originalModes = MapboxDraw.modes;
+var drawConstants;
+if (typeof MapboxDraw.constants !== 'undefined') {
+    drawConstants = MapboxDraw.constants
+} else {
+    drawConstants = await import('@mapbox/mapbox-gl-draw/src/constants');
+}
```

Comment:
Dynamic import to dev dependencies doesn't work after upgrading the library.

Reply:
This dynamic import is there in order to keep backwards compatibility and make the repo work for 1.3 and >1.4 at once.I can leave only the 1.4 version

Reply:
I'm sorry, but this still doesn't compile. `npm run dev` throws `[!] RollupError: Module format "umd" does not support top-level await. Use the "es" or "system" output formats rather.`